packages/@coorpacademy-components/src/molecule/card/index.js

```diff
@@ -4,15 +4,70 @@ import classnames from 'classnames';
 import {get, isEmpty, isUndefined, pick} from 'lodash/fp';
 import {
   NovaSolidLocksLock11 as LockIcon,
-  NovaCompositionCoorpacademyPictures as PicturesIcon
+  NovaCompositionCoorpacademyPictures as PicturesIcon,
+  NovaCompositionCoorpacademyScorm as ScormIcon,
+  NovaCompositionCoorpacademyArticle as ArticleIcon,
+  NovaCompositionCoorpacademyVideo as VideoIcon
 } from '@coorpacademy/nova-icons';
+import isExternalContent from '../../util/external-content';
 import Provider from '../../atom/provider';
 import CardContentInfo, {MODES} from '../card-content';
 import Customer from './customer';
 import Favorite from './favorite';
 import Notification from './notification';
 import style from './style.css';
 
+const ICONS = {
+  scorm: {icon: ScormIcon, color: '#FFB800'},
+  article: {icon: ArticleIcon, color: '#365FCD'},
+  video: {icon: VideoIcon, color: '#E8335E'}
+};
+
+const CardBackground = ({type, image, empty}, context) => {
+  const {skin} = context;
+  const primaryColor = get('common.primary', skin);
+  const whiteColor = get('common.white', skin);
+  const externalContent = isExternalContent(type);
+
+  if (externalContent) {
+    const IconType = ICONS[type].icon;
+    const IconColor = ICONS[type].color;
```

Comment:
camel case ?

Reply:
btw, `ICONS[type]` can return undefined in case of using undefined type in `ICONS` const, and so, accessing icon or color properties can throw errors

Reply:
done, j'ai modifié la condition du if